etc/picongpu/jureca-jsc/batch_picongpu.profile.example

```diff
@@ -0,0 +1,109 @@
+# Name and Path of this Script ############################### (DO NOT change!)
+export PIC_PROFILE=$(cd $(dirname $BASH_SOURCE) && pwd)"/"$(basename $BASH_SOURCE)
+
+# User Information ######################################### (edit those lines)
+#   - automatically add your name and contact to output file meta data
+#   - send me a mail on batch system jobs: NONE, BEGIN, END, FAIL, REQUEUE, ALL,
+#     TIME_LIMIT, TIME_LIMIT_90, TIME_LIMIT_80 and/or TIME_LIMIT_50
+export MY_MAILNOTIFY="NONE"
+export MY_MAIL="someone@example.com"
+export MY_NAME="$(whoami) <$MY_MAIL>"
+
+# Project Information ######################################## (edit this line)
+#   - project account for computing time
+export proj=$(groups | awk '{print $5}')
```

Comment:
maybe there is a variable on this system already which we can use? Just in case, that would be more robust if they provide one. just had to do this on systems that don't export one. (same in other profiles below)

Reply:
I've now looked again for such a variable, but did not find it. However, I've now found their script that sets some other useful env variables (like `$SCRATCH` that we've set by hand), will include that.